extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/serde/package.scala

```diff
@@ -82,4 +91,33 @@ package object serde {
       .map(s => s.operationType)
       .getOrElse(QUERY)
   }
+
+  /**
+   * get extractor instance by extractor class name
+   * @param extractorKey explicitly load extractor by its simple class name.
+   *                           null by default means get extractor by extractor class.
+   * @param ct class tag of extractor class type
+   * @tparam T extractor class type
+   * @return
+   */
+  def lookupExtractor[T <: Extractor](extractorKey: String = null)(
```

Comment:
`extractorKey: String = null` => `extractorKey: Option[String] = None`

Reply:
Adopted.

Reply:
Made `extractorKey` not null by default.